.github/workflows/consul.aspnetcore.yml

```diff
@@ -0,0 +1,66 @@
+name: Consul | AspNetCore
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  build:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
+    name: Consul | AspNetCore
+    runs-on: windows-latest
+
+    steps:
+      - name: Consul | AspNetCore | Checkout
+        uses: actions/checkout@v2
+
+      - name: Consul | AspNetCore | .NET Core
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 3.1.300
+
+      - name: Consul | AspNetCore | Build
+        run: dotnet build -c=Release -p:TreatWarningsAsErrors=true
+
+      - name: Consul | AspNetCore | Tests
+        run: |
+          cd Consul.AspNetCore.Test
+          dotnet test -c=Release --no-build -v=Normal
+          cd ..
+
+      - name: Consul | AspNetCore | Package
+        run: |
+          if ("${{ github.ref }}" -like "refs/tags/v*") {
+            $tag = "${{ github.ref }}".SubString(11)
+            $version = (Select-Xml -path Consul.AspNetCore/Consul.AspNetCore.csproj -XPath "/Project/PropertyGroup/VersionPrefix/text()").node.Value
+            if (-not ($tag -eq $version)) {
+              echo "There is a mismatch between the project version ($version) and the tag ($tag)"
+              exit 1
+            }
```

Comment:
We cannot use the same tag name pattern for both packages (`Consul` and `Consul.AspNetCore`) as the version validation logic would be broken. Either  `Consul` or `Consul.AspNetCore` would always have "wrong" version number.

I think we should use a more specific prefix here, e.g. `refs/tags/Consul.AspNetCore_v*`